content/blog/2024/202401-review.rst

```diff
@@ -0,0 +1,68 @@
+================================================================
+sciwork January 2024 review: weekly meetup and Hsinchu scisprint
+================================================================
+
+:date: 2024-03-04
+:author: Ming-Jie Kuo
+:category: blog
+
+Weekly Meetup
+-------------
+In the January meetup, Yung-Yu introduced the CESE method, which is utilized in
+the modmesh project to solve the fluid dynamics problem. This provided modmesh
```

Comment:
You may add the link to the modmesh repository as a passive way to promote, if you want.

Reply:
I added a link to the modmesh repository.